.github/workflows/node-tests.yml

```
@@ -0,0 +1,45 @@
+name: Node Tests
+
+on:
+  workflow_call:
+    inputs:
+      node:
+        default: 16
+        required: false
+        type: number
+
+jobs:
+  npm-ci:
+    name: Install, build, and test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs of this workflow (pull requests only)
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+      - name: Checkout code
+        uses: actions/checkout@v2
```

This is the only place where you are using `actions/checkout@v2`.

```suggestion
        uses: actions/checkout@v3
```